src/main/java/at/gammastrahlung/monopoly_server/game/Game.java

```diff
@@ -245,4 +249,66 @@ public enum GameState {
          */
         ENDED
     }
+
+    // Starts a new auction for a specific property
+    public void startAuction(Property auctionProperty) {
+        currentAuction = new AuctionSystem(auctionProperty, new AuctionSystem.AuctionEventListener() {
+            @Override
+            public void onBidUpdated(Player player, int newBid) {
+                // Implement what should happen when a bid is updated
+                System.out.println("Bid updated: " + player.getName() + " bids " + newBid);
+            }
+
+            @Override
+            public void onAuctionFinalized(Player winner, int winningBid) {
+                // Implement what should happen when the auction is finalized
+                System.out.println("Auction finalized: " + winner.getName() + " wins with " + winningBid);
+            }
+        });
+    }
+
+    // Allows a player to place a bid
+    public boolean placeBid(Player player, int bid) {
+        if (currentAuction != null) {
+            return currentAuction.placeBid(player, bid);
+        }
+        System.out.println("No active auction to place bid.");
+        return false;
+    }
+
+    // Ends the current auction and transfers ownership
+    public void finalizeAuction() {
+        if (currentAuction != null) {
+            Player winner = currentAuction.finalizeAuction();
+            if (winner != null) {
+                System.out.println("Auction won by: " + winner.getName() + " with a bid of: " + currentAuction.getHighestBid());
+            } else {
+                System.out.println("Auction closed with no valid bids.");
```

Comment:
Don't output something in the Game class.
There is no way for the players to read this